src/_gettsim/groupings.py

```diff
@@ -76,7 +76,7 @@ def fg_id_numpy(  # noqa: PLR0913
 
         p_id_to_fg_id[current_p_id] = next_fg_id
 
-        current_vg_id = hh_id[index]
+        current_hh_id = hh_id[index]
```

Comment:
I replaced `vg` with `hh` in the groupings function as well. That's correct, right?

Reply:
Correct. VG should go completely, including any reference to it in the docs etc.